src/constants.py

```diff
@@ -27,5 +27,10 @@
 CHARMED_MYSQL_COMMON_DIRECTORY = "/var/snap/charmed-mysql/common"
 MYSQLD_SOCK_FILE = f"{CHARMED_MYSQL_COMMON_DIRECTORY}/mysql/mysqld.sock"
 MYSQLD_CONFIG_DIRECTORY = f"{CHARMED_MYSQL_COMMON_DIRECTORY}/mysql/conf.d"
+MYSQLD_DEFAULTS_CONFIG_FILE = f"{CHARMED_MYSQL_COMMON_DIRECTORY}/mysql/mysql.cnf"
```

Comment:
hm, can we follow snap structure according to the spec https://docs.google.com/document/d/1MyT4Wt_-vgPwhwEjC-oeHRdLYrZWaQzFGTwclT7hD1U/edit

Reply:
@jardon mentioned that we will be working on restructuring the charmed-mysql snap next pulse. captured the effort in [dpe-1459](https://warthogs.atlassian.net/browse/DPE-1459)